app/screens/ProductDetailsScreen/ProductDetailsScreen.tsx

```diff
@@ -0,0 +1,143 @@
+import { Header } from '../../components/Header/Header';
+import { IconBasket } from '../../components/icons/IconBasket';
+import { IconHeart } from '../../components/icons/IconHeart';
+import { IconArrowLeft } from '../../components/icons/IconArrowLeft';
+
+export const ProductDetailsScreen: React.FC = (): JSX.Element => {
+    const [refreshing, setRefreshing] = useState(false);
+    const product = {
+        attributes: {
+            price: '86.99',
+            description: 'Doloremque ab accusamus blanditiis et. Deleniti ma…oloribus ad. Non repellat maiores delectus rerum.',
+            name: 'Raw Edge T Shirt'
+        },
+        id: '16'
+    };
+
+    const wait = (timeout: number) => {
+        return new Promise(resolve => setTimeout(resolve, timeout));
+    }
+
+    const onRefresh = React.useCallback(() => {
+        setRefreshing(true);
+        wait(2000).then(() => setRefreshing(false));
+    }, []);
+
+    return (
+    <SafeAreaView>
+        <ScrollView
+            refreshControl={
+            <RefreshControl
+                refreshing={refreshing}
+                onRefresh={onRefresh}
+            />
+        }>
+            <Header>
+                <View>
+                    <IconArrowLeft fill={'#FFFFFF'} />
+                </View>
+                <View style={styles.headerIcons}>
+                    <IconHeart fill={'#FFFFFF'} />
+                    <View style={styles.headerIconRight}>
+                        <IconBasket fill={'#FFFFFF'} />
+                    </View>
+                </View>
+            </Header>
+            <View style={styles.container}>
+                <View style={styles.imgContainer}>
+                    <Image
+                        style={styles.img}
+                        source={{uri: `https://picsum.photos/id/${product.id}/250`}}
+                        resizeMode={'contain'}
+                    />
+                </View>
+                <View style={styles.cell}>
+                    <Text style={styles.cellValue}>{product.attributes.name}</Text>
+                    <Text style={styles.cellTitle}>{product.attributes.price}</Text>
+                </View>
+                <View style={styles.cell}>
+                    <Text style={styles.cellTitle}>Select color</Text>
+                    <View style={styles.btn}>
+                        <Button
+                            title="Blue"
+                            disabled
+                            onPress={() => Alert.alert('Cannot press this one')}
+                        />
+                    </View>
+                </View>
+                <View style={styles.cell}>
+                    <Text style={styles.cellTitle}>Description</Text>
+                    <Text style={styles.cellValue}>{product.attributes.description}</Text>
+                </View>
+            </View>
+        </ScrollView>
```

Comment:
<img width="411" alt="Screenshot 2022-01-05 at 14 27 56" src="https://user-images.githubusercontent.com/58725560/148210464-23292b7c-ec70-4f24-a8d8-028eef1687ad.png">
 
I can't read your description completely. Consider having bottom margin as you have absolute positioned button there